frontend/public/tinymce/langs/zh_TW.js

```diff
@@ -0,0 +1,419 @@
+"Invert": "\u53cd\u8f49",
+"Apply": "\u61c9\u7528",
+"Back": "\u5f8c\u9000",
+"Insert date\/time": "\u63d2\u5165 \u65e5\u671f\/\u6642\u9593",
+"Date\/time": "\u65e5\u671f\/\u6642\u9593",
+"Insert\/Edit Link": "\u63d2\u5165\/\u7de8\u8f2f\u9023\u7d50",
+"Insert\/edit link": "\u63d2\u5165\/\u7de8\u8f2f\u9023\u7d50",
+"Text to display": "\u986f\u793a\u6587\u5b57",
+"Url": "\u7db2\u5740",
+"Open link in...": "\u958b\u555f\u9023\u7d50\u65bc...",
+"Current window": "\u76ee\u524d\u8996\u7a97",
+"None": "\u7121",
+"New window": "\u53e6\u958b\u8996\u7a97",
+"Remove link": "\u79fb\u9664\u9023\u7d50",
+"Anchors": "\u52a0\u5165\u9328\u9ede",
+"Link...": "\u9023\u7d50...",
+"Paste or type a link": "\u8cbc\u4e0a\u6216\u8f38\u5165\u9023\u7d50",
+"The URL you entered seems to be an email address. Do you want to add the required mailto: prefix?": "\u4f60\u6240\u586b\u5beb\u7684URL\u70ba\u96fb\u5b50\u90f5\u4ef6\uff0c\u9700\u8981\u52a0\u4e0amailto:\u524d\u7db4\u55ce\uff1f",
+"The URL you entered seems to be an external link. Do you want to add the required http:\/\/ prefix?": "\u4f60\u6240\u586b\u5beb\u7684URL\u5c6c\u65bc\u5916\u90e8\u93c8\u63a5\uff0c\u9700\u8981\u52a0\u4e0ahttp:\/\/:\u524d\u7db4\u55ce\uff1f",
+"Link list": "\u9023\u7d50\u6e05\u55ae",
+"Insert video": "\u63d2\u5165\u5f71\u97f3",
+"Insert\/edit video": "\u63d2\u4ef6\/\u7de8\u8f2f \u5f71\u97f3",
+"Insert\/edit media": "\u63d2\u5165\/\u7de8\u8f2f \u5a92\u9ad4",
+"Alternative source": "\u66ff\u4ee3\u5f71\u97f3",
+"Alternative source URL": "\u66ff\u4ee3\u4f86\u6e90URL",
+"Media poster (Image URL)": "\u5a92\u9ad4\u6d77\u5831\uff08\u5f71\u50cfImage URL\uff09",
+"Paste your embed code below:": "\u8acb\u5c07\u60a8\u7684\u5d4c\u5165\u5f0f\u7a0b\u5f0f\u78bc\u8cbc\u5728\u4e0b\u9762:",
+"Embed": "\u5d4c\u5165\u78bc",
+"Media...": "\u5a92\u9ad4...",
+"Nonbreaking space": "\u4e0d\u5206\u884c\u7684\u7a7a\u683c",
+"Page break": "\u5206\u9801",
+"Paste as text": "\u4ee5\u7d14\u6587\u5b57\u8cbc\u4e0a",
+"Preview": "\u9810\u89bd",
+"Print...": "\u5217\u5370...",
+"Save": "\u5132\u5b58",
+"Find": "\u641c\u5c0b",
+"Replace with": "\u66f4\u63db",
+"Replace": "\u66ff\u63db",
+"Replace all": "\u66ff\u63db\u5168\u90e8",
+"Previous": "\u4e0a\u4e00\u500b",
+"Next": "\u4e0b\u4e00\u500b",
+"Find and replace...": "\u5c0b\u627e\u53ca\u53d6\u4ee3...",
+"Could not find the specified string.": "\u7121\u6cd5\u67e5\u8a62\u5230\u6b64\u7279\u5b9a\u5b57\u4e32",
+"Match case": "\u76f8\u5339\u914d\u6848\u4ef6",
+"Find whole words only": "\u50c5\u627e\u51fa\u5b8c\u6574\u5b57\u532f",
+"Spell check": "\u62fc\u5beb\u6aa2\u67e5",
+"Ignore": "\u5ffd\u7565",
+"Ignore all": "\u5ffd\u7565\u6240\u6709",
+"Finish": "\u5b8c\u6210",
+"Add to Dictionary": "\u52a0\u5165\u5b57\u5178\u4e2d",
+"Insert table": "\u63d2\u5165\u8868\u683c",
+"Table properties": "\u8868\u683c\u5c6c\u6027",
+"Delete table": "\u522a\u9664\u8868\u683c",
+"Cell": "\u5132\u5b58\u683c",
+"Row": "\u5217",
+"Column": "\u884c",
+"Cell properties": "\u5132\u5b58\u683c\u5c6c\u6027",
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/dataease/dataease/pullRequest?prid=7687804)
```suggestion
'Cell properties': "\u5132\u5b58\u683c\u5c6c\u6027",
```